src/components/HypothesisTesting/SimulateTypeOneError.js

```diff
@@ -68,6 +68,7 @@ export default function SimulateTypeOneError({ popShape, mu0, alpha, distType, s
         const sample2 = (testType === "twoSample") ? _.sampleSize(population2, size) : [];
         const sampleMean2 = populationMean(sample2);
         const testStatistic = (testType === "oneSample")
+        const meansDiff = 0
```

Comment:
Watch out, this is now in the middle of a ternary expression that calculates `testStatistic`.

Actually, we shouldn't even need this expression anymore, I believe.